src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Converters/Value/EnumConverter.cs

```diff
@@ -266,64 +216,49 @@ public override void Write(Utf8JsonWriter writer, T value, JsonSerializerOptions
                     writer.WriteNumberValue(Unsafe.As<T, sbyte>(ref value));
                     break;
                 default:
-                    ThrowHelper.ThrowJsonException();
+                    Debug.Fail("Should not be reached");
                     break;
             }
         }
 
         internal override T ReadAsPropertyNameCore(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
         {
-#if NET
-            if (TryParseEnumCore(ref reader, out T value))
-#else
-            string? enumString = reader.GetString();
-            if (TryParseEnumCore(reader.GetString(), out T value))
-#endif
+            // NB JsonSerializerOptions.DictionaryKeyPolicy is ignored on deserialization.
+            // This is true for all converters that implement dictionary key serialization.
```

Comment:
What does NB mean?

Reply:
It's abbreviation for [Nota bene](https://en.wikipedia.org/wiki/Nota_bene)